src/Model/User.php

```diff
@@ -168,6 +172,18 @@ function( $cap ) {
 				'nicename'          => function() {
 					return ! empty( $this->data->user_nicename ) ? $this->data->user_nicename : null;
 				},
+				'status'          => function() {
+					return ! empty( $this->data->user_status ) ? $this->data->user_status : null;
+				},
+				'activation_key'          => function() {
+					return ! empty( $this->data->user_activation_key ) ? $this->data->user_activation_key : null;
+				},
+				'pass'          => function() {
```

Comment:
So we will need to make sure we're handling cap checks properly on all these fields. 

For example, the password should only be exposed to the current user (or users with proper caps to see it. . .if that's a thing? I don't think anyone _but_ the current user _can_ see a Password, eh? Even admins?

Reply:
Funny thing is WP does not store actual passwords. They hash the information.

> This is why WordPress passwords are hashed: The only reason the stored value needs to be used for is to compare whether the password that the user enters on next login matches what was originally stored as the password. WordPress never displays the password anywhere, so it never needs to be able to decrypt it. And because there is no need for that, it’s safer to use a hashing function which basically makes it irreversible.
Ref: https://felix-arntz.me/blog/storing-confidential-data-in-wordpress/

Should we still work to query this?

Reply:
Ya, I'm actually thinking we don't need to expose passwords at all. 

My initial thought was that you could build a UI to edit your user profile and you'd be able to see your current password in a `<input type="password" value="{user.password}" />` field, but it looks like even WordPress core doesn't show a password field on your profile, the only option is to generate a new password. 

So ya, I think we can leave password out altogether.

Reply:
I also think passwords shouldn't be exposed in the schema. FWIW we explicitly remove the user_pass from the user object passed in to the model as an extra security measure. This way the data isn't being passed around in filters and such https://github.com/wp-graphql/wp-graphql/blob/dbdf14f5b6adb12d8343f9d3b69cb7a96c7a3b3c/src/Model/User.php#L51-L52